client/src/components/TeamAlphaDetails/TeamAlphaDetails.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import DeveloperProfile from "../DeveloperProfile/DeveloperProfile";
+import "./TeamAlphaDetails.css";
+
+const teamAlphaData = [
+  {
+    name: "Saghar Hosseinardi",
+    role: "Full-Stack Developer",
+    githubUsername: "SagharHosseinmardi",
+    linkedinUsername: "Saghar Hosseinmardi",
```

Comment:
I wonder if we should add an 'about' section after the linkedin bit? that will have just a few lines about each of us-something light and fun? I could do this..... what do you think?

Reply:
yes it would be great can you do it please?

Reply:
Ok. I will do that once I get home today...